docs/src/index.md

```diff
@@ -26,6 +26,28 @@ They should also have a test-only dependency on [ChainRulesTestUtils.jl](https:/
 
 Note that the packages with rules do not have to depend on AD systems, and neither do the AD systems have to depend on individual packages.
 
+## ChainRules roll-out status
+
+Approximately [70 packages](https://juliahub.com/ui/Packages/ChainRulesCore/G6ax7/1.11.1?page=2) depends on ChainRulesCore to define rules.
+
+6 AD engines use ChainRules to get access to rules:
+
+[Zygote.jl](https://github.com/FluxML/Zygote.jl) is a reverse-mode AD that supports using `rrule`s, calling back into AD, and opting out of rules.
+However, if defined, its own primitives (`@adjoint`s) take precedence before `rrule`s.
+Internally it uses its own set of tangent types, e.g. `nothing` instead of `NoTangent`/`ZeroTangent`.
+It also `unthunk`s every tangent.
+
+[Diffractor.jl](https://github.com/JuliaDiff/Diffractor.jl) is a forward- and reverse-mode AD that fully supports ChainRules, including calling back into AD, opting out of rules, and uses tangent types internally.
+
+[Yota](https://github.com/dfdx/Yota.jl) is a reverse-mode AD that supports using `rrule`s, but not calling back into AD and opting out of rules.
```

Comment:
Yota's support is actually pretty comprehensive.
```suggestion
[Yota](https://github.com/dfdx/Yota.jl) is a reverse-mode AD that fully supports ChainRules, including calling back into AD, opting out of rules, and uses tangent types internally.
```

Reply:
Oh, interesting! I took a quick look and couldn't find a `RuleConfig` defined anywhere so just assumed that it doesn't support that.

Reply:
It's there